### ISSUE TITLE
SIGABRT in draco::MeshPredictionSchemeConstrainedMultiParallelogramDecoder<int, draco::PredictionSchemeWrapDecodingTransform<int, int>, draco::MeshPredictionSchemeData<draco::MeshAttributeCornerTable> >::ComputeOriginalValues

### ISSUE THREAD

**Issue by najadojo** [google/draco#397]
Using american fuzzy lop 2.49b on draco_decoder I found a crash at the stack:

```
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff713af5d in __GI_abort () at abort.c:90
#2  0x00007ffff7b084e3 in __gnu_debug::_Error_formatter::_M_error() const ()
   from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x000055555558aabd in std::__debug::vector<bool, std::allocator<bool> >::operator[] (this=0x555555962f78, __n=0)
    at /usr/include/c++/7/debug/vector:417
#4  0x000055555565c1bb in draco::MeshPredictionSchemeConstrainedMultiParallelogramDecoder<int, draco::PredictionSchemeWrapDecodingTransform<int, int>, draco::MeshPredictionSchemeData<draco::MeshAttributeCornerTable> >::ComputeOriginalValues (this=0x555555962eb0, in_corr=0x555555963150, out_data=0x555555963150, num_components=28)
    at /home/jamie/draco/src/draco/compression/attributes/prediction_schemes/mesh_prediction_scheme_constrained_multi_parallelogram_decoder.h:155
#5  0x00005555556530b3 in draco::SequentialIntegerAttributeDecoder::DecodeIntegerValues (this=0x555555955eb0, 
    point_ids=std::__debug::vector of length 703, capacity 703 = {...}, in_buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/attributes/sequential_integer_attribute_decoder.cc:141
#6  0x0000555555652bf9 in draco::SequentialIntegerAttributeDecoder::DecodeValues (this=0x555555955eb0, 
    point_ids=std::__debug::vector of length 703, capacity 703 = {...}, in_buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/attributes/sequential_integer_attribute_decoder.cc:60
#7  0x00005555556791a7 in draco::SequentialAttributeDecoder::DecodePortableAttribute (this=0x555555955eb0, 
    point_ids=std::__debug::vector of length 703, capacity 703 = {...}, in_buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/attributes/sequential_attribute_decoder.cc:41
#8  0x000055555565029a in draco::SequentialAttributeDecodersController::DecodePortableAttributes (
    this=0x555555955b40, in_buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/attributes/sequential_attribute_decoders_controller.cc:66
#9  0x000055555562d678 in draco::AttributesDecoder::DecodeAttributes (this=0x555555955b40, in_buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/attributes/attributes_decoder.h:57
#10 0x0000555555650218 in draco::SequentialAttributeDecodersController::DecodeAttributes (this=0x555555955b40, 
    buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/attributes/sequential_attribute_decoders_controller.cc:59
#11 0x00005555555b18bf in draco::PointCloudDecoder::DecodeAllAttributes (this=0x55555593b230)
    at /home/jamie/draco/src/draco/compression/point_cloud/point_cloud_decoder.cc:151
#12 0x00005555555b178e in draco::PointCloudDecoder::DecodePointAttributes (this=0x55555593b230)
    at /home/jamie/draco/src/draco/compression/point_cloud/point_cloud_decoder.cc:141
#13 0x00005555555b1258 in draco::PointCloudDecoder::Decode (this=0x55555593b230, options=..., 
    in_buffer=0x7fffffffda70, out_point_cloud=0x55555593b050)
    at /home/jamie/draco/src/draco/compression/point_cloud/point_cloud_decoder.cc:99
#14 0x0000555555570fd1 in draco::MeshDecoder::Decode (this=0x55555593b230, options=..., in_buffer=0x7fffffffda70, 
    out_mesh=0x55555593b050) at /home/jamie/draco/src/draco/compression/mesh/mesh_decoder.cc:24
#15 0x000055555556c7e4 in draco::Decoder::DecodeBufferToGeometry (this=0x7fffffffdab0, in_buffer=0x7fffffffda70, 
    out_geometry=0x55555593b050) at /home/jamie/draco/src/draco/compression/decode.cc:121
#16 0x000055555556c169 in draco::Decoder::DecodeMeshFromBuffer (this=0x7fffffffdab0, in_buffer=0x7fffffffda70)
    at /home/jamie/draco/src/draco/compression/decode.cc:86
```

./draco_decoder -i [id:000402,sig:11,src:002825,op:int16,pos:2799,val:+0](https://github.com/google/draco/files/2027689/id.000402.sig.11.src.002825.op.int16.pos.2799.val.0.gz)

**Comment by najadojo** [comment]
I've identified more files with similar but unique stacks:

[id:000183,sig:11,src:000208,op:flip1,pos:111.gz](https://github.com/google/draco/files/2028380/id.000183.sig.11.src.000208.op.flip1.pos.111.gz)
[id:000161,sig:11,src:000026,op:arith8,pos:922,val:-26.gz](https://github.com/google/draco/files/2028381/id.000161.sig.11.src.000026.op.arith8.pos.922.val.-26.gz)
[id:000114,sig:11,src:000009,op:int16,pos:2796,val:-32768.gz](https://github.com/google/draco/files/2028383/id.000114.sig.11.src.000009.op.int16.pos.2796.val.-32768.gz)
[id:000106,sig:11,src:000009,op:int16,pos:582,val:be:+512.gz](https://github.com/google/draco/files/2028384/id.000106.sig.11.src.000009.op.int16.pos.582.val.be.512.gz)
[id:000101,sig:11,src:000009,op:arith8,pos:440,val:-13.gz](https://github.com/google/draco/files/2028385/id.000101.sig.11.src.000009.op.arith8.pos.440.val.-13.gz)
[id:000182,sig:11,src:000207,op:havoc,rep:2.gz](https://github.com/google/draco/files/2028390/id.000182.sig.11.src.000207.op.havoc.rep.2.gz)
[id:000108,sig:11,src:000009,op:int16,pos:2795,val:+1.gz](https://github.com/google/draco/files/2028391/id.000108.sig.11.src.000009.op.int16.pos.2795.val.1.gz)

**Comment by ondys** [comment]
fixed in 1.3.2
